intera_examples/scripts/camera_display.py

```diff
@@ -62,24 +62,49 @@ def main():
         '-c', '--camera', type=str, default="head_camera",
         choices=valid_cameras, help='Setup Camera Name for Camera Display')
     parser.add_argument(
-        '-r', '--raw', action='store_true', 
+        '-r', '--raw', action='store_true',
         help='Specify use of the raw image (unrectified) topic')
     parser.add_argument(
         '-e', '--edge', action='store_true',
         help='Streaming the Canny edge detection image')
+    parser.add_argument(
+        '-g', '--gain', type=int,
+        help='Set gain for camera ([0-79], -1 = auto)')
+    parser.add_argument(
+        '-x', '--exposure', type=int,
+        help='Set exposure for camera ([0-100], -1 = auto)')
     args = parser.parse_args()
 
     print("Initializing node... ")
     rospy.init_node('camera_display', anonymous=True)
-    camera = intera_interface.Cameras()
-    if not camera.verify_camera_exists(args.camera):
+    cameras = intera_interface.Cameras()
+    if not cameras.verify_camera_exists(args.camera):
         rospy.logerr("Invalid camera name, exiting the example.")
         return
-    camera.start_streaming(args.camera)
+    cameras.start_streaming(args.camera)
     rectify_image = not args.raw
     use_canny_edge = args.edge
-    camera.set_callback(args.camera, show_image_callback,
+    cameras.set_callback(args.camera, show_image_callback,
         rectify_image=rectify_image, callback_args=(use_canny_edge, args.camera))
+    camera_interface = cameras.cameras_io[args.camera]['interface']
```

Comment:
This section seems like it could be inside the Cameras interface itself, rather than just the example. Thoughts?